ClientApp/viking-vault/src/Dashboard.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {constants} from "./ConstantVariables";
+import { Redirect } from "react-router-dom";
+import UserPage from "./components/UserPage";
+import { AdminPage } from "./display_user_list/AdminPage";
+
+const baseUrl = constants.baseUrl;
```

Comment:
if you already have the baseUrl stored in constants, don't create a new constant, use that one